cognito-post-authentication/src/main/java/no/unit/nva/cognito/model/Role.java

```diff
@@ -17,4 +19,23 @@ public Role(@JsonProperty("rolename") String rolename) {
     public String getRolename() {
         return rolename;
     }
+
+    @Override
```

Comment:
We should consider publishing the DTOs so that we do not have to have duplicates